src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs

```diff
@@ -480,7 +480,7 @@ private NuGetPackageManager GetNuGetPackageManager(string solutionDirectory)
             {
                 if (!string.IsNullOrEmpty(exception.Message))
                 {
-                    nuGetProjectContext.Log(MessageLevel.Warning, exception.Message);
+                    nuGetProjectContext.Log(MessageLevel.Error, exception.Message);
```

Comment:
Warnings and errors should always have an NU code, so that we can write docs, and it's easier for customers to learn how to resolve the problem.

Reply:
@zivkan 
Since it wide variety of exception can be caught here, so hard to tell exactly what went wrong. Could it be just [NU1000](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1000) error? Please advise.

Reply:
Yes, NU1000 is better than nothing. Customers will hopefully  follow the doc's advice and file bug for us to make more specific error codes, so we can write docs for the different scenarios.

Reply:
https://github.com/NuGet/NuGet.Client/pull/4248#discussion_r713295746